src/clj/fluree/db/indexer/default.cljc

```diff
@@ -510,15 +502,15 @@
 (defn do-index
   "Performs an index operation and returns a promise-channel of the latest db once complete"
   [indexer {:keys [t branch] :as db} {:keys [update-commit changes-ch] :as opts}]
-  (let [[lock? index-state] (lock-indexer (:state-atom indexer) branch t update-commit)
+  (let [[lock? index-state]   (lock-indexer (:state-atom indexer) branch t update-commit)
         {:keys [tempid port]} index-state]
     (if lock?
       ;; when we have a lock, reindex and put updated db onto pc.
       (go
         (try*
           (push-index-event indexer :index-start index-state)
-          (let [indexed-db   (<? (refresh indexer db opts))
-                index-state* (unlock-indexer (:state-atom indexer) branch tempid indexed-db)
+          (let [indexed-db                      (<? (refresh indexer db opts))
+                index-state*                    (unlock-indexer (:state-atom indexer) branch tempid indexed-db)
```

Comment:
yeah, i agree. I find it annoying too, but emacs just does it. i know i can configure this because you can configure everything in emacs. the question is whether i can figure out how to configure this in less than an hour...

in any case, i'll take this one out by hand.

Reply:
I usually just enter a line of whitespace between the destructuring form and the rest, do a quick `C-c SPC`, and then remove the whitespace. Or not remove the whitespace, depends.

Reply:
yeah, i do that too sometimes, but i also don't really like needing to enter that empty line just because there's a huge destructuring form. it isn't the worst though.